core/src/test/java/org/opentosca/toscana/core/BaseSpringTest.java

```diff
@@ -1,45 +1,64 @@
 package org.opentosca.toscana.core;
 
+import java.io.File;
+import java.io.IOException;
+
+import javax.annotation.PostConstruct;
+
+import org.opentosca.toscana.core.testutils.CategoryAwareSpringRunner;
+import org.opentosca.toscana.core.util.Preferences;
+
 import org.apache.commons.io.FileUtils;
 import org.junit.After;
 import org.junit.Before;
 import org.junit.runner.RunWith;
-import org.opentosca.toscana.core.testutils.CategoryAwareSpringRunner;
-import org.opentosca.toscana.core.util.Preferences;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
 import org.springframework.test.annotation.DirtiesContext;
-import org.springframework.test.context.ActiveProfiles;
 import org.springframework.test.context.ContextConfiguration;
 import org.springframework.test.context.TestPropertySource;
 
-import java.io.IOException;
-
 /**
  * Extend from this class in order to inherit important configurations Sets up Spring Test Context regarding to the Test
- * Configuration After every test method, refreshes the context. After every test method, deletes written files from
- * disk
+ * Configuration After every test method, refreshes the context. 
+ *
+ * If disk access is needed, use {@field #tmpdir}
  */
 @RunWith(CategoryAwareSpringRunner.class)
 @ContextConfiguration(classes = {TestCoreConfiguration.class})
 @DirtiesContext(classMode = DirtiesContext.ClassMode.AFTER_EACH_TEST_METHOD)
-@ActiveProfiles("dummy_plugins")
 @Component
```

Comment:
I honestly dont think we need this here, maybe run it without the annotation and see if its works

Reply:
you're right, removing `@Component`